### PR TITLE
Wrap diagnostic output into collapsed <details> block

### LINKED ISSUE #4244: ddev issue template: Put diagnostic information inline into issues and hide under "details" collapsed block
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

When reporting an issue with ddev, I need to create a gist and paste link to it.

### Describe your solution

Instead will be much easier to paste the full text into the issue creation form, and the issue template will hide (collapse) this text under the `<details>` block. Here is an example of this: https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab

### Describe alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MurzNN**
## The Issue
We need to create a gist now to paste diagnostic information, that is too many actions for a regular user.

## How This PR Solves The Issue
This fix should add the pre-filled text to the diagnostic field.

## Manual Testing Instructions
Create a new issue and fill the diagnostic data.

## Related Issue Link(s)
Fixes #4244

## Release/Deployment Notes

<a href="https://gitpod.io/#https://github.com/ddev/ddev/pull/4865"></a>

### CODE REVIEW

**Comment by rfay** [comment]
IIRC this approach doesn't work in bug report template, but please try it in your own repo or fork to make sure.

**Comment by MurzNN** [comment]
There was a whitespace problem, fixed now, here is the example bug: https://github.com/MurzNN/ddev/issues/7
Please check.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -11,7 +11,15 @@ body:
           required: true
   - type: textarea
     attributes:
-      label: Run a Diagnostic and Paste Link Here
+      label: Diagnostic information
+      value: |
+        <details><summary>Expand diagnostic information</summary>
+        
+        ```
+        [COPY-PASTE HERE THE DIAGNOSIC INFORMATION]
```
Comment (rfay) [inline .github/ISSUE_TEMPLATE/bug_report.yml:19]:
```suggestion
        [COPY-PASTE HERE THE OUTPUT OF `ddev debug test`]
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -11,7 +11,15 @@ body:
...
+        ```
+        
+        </details>         
       description: |
         If you’re having trouble with DDEV, help us with context that will skip a bunch of questions:
```
Comment (rfay) [inline .github/ISSUE_TEMPLATE/bug_report.yml:24]:
```suggestion
        Whether you’re having trouble or not, help us with context that will skip a bunch of questions we'd have to ask afterwards:
```

**Review by rfay — CHANGES_REQUESTED** [review]
Thanks, I think this is great. Minor suggestions you can accept (and preferably test in your fork too). I appreciate it. I think the last time this was attempted it didn't work out.

**Review by mattstein — APPROVED** [review]

**Pull request merged**